vmm/src/default_syscalls/aarch64.rs

```diff
@@ -69,7 +68,6 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
             ),
             allow_syscall(SYS_mmap),
             allow_syscall(libc::SYS_munmap),
-            #[cfg(target_env = "musl")]
```

Comment:
On x86_64 SYS_openat was only used with musl while SYS_open was used with glibc. Is this different on aarch64?